content/c/concepts/basic-output/basic-output

```diff
@@ -0,0 +1,71 @@
+---
+Title: "Basic Output"
+Subjects:
+  - "Code Foundations"
+  - "Computer Science"
+Tags:
+  - "Print"
+  - "Output"
+---
+
+In the C language basic console output is achieved using functions from the standard input output header file (stdio.h)
+
+## Including Output functions
+
+C itself has no ability to output any information, in order to so the stdio header file must be included in the program, which is typically done at the top of the file.
+
+```c
+#include <stdio.h>
+```
+
+Once this is done the functions included in it can be used.
+
+## Printing Single Characters With `putchar`
```

Comment:
We usually write the parentheses when talking function names at Codecademy:

`putchar()`

Reply:
Done